docs/user/dashboard/dashboard.asciidoc

```diff
@@ -4,505 +4,186 @@
 
-* Create complex visualizations
-* Use JSON and a different syntax for declaring visualizations
-* Are not fully interchangeable
+. Click *Save*.
 
-For more information about *Vega* and *Vega-Lite*, refer to:
+[[share-the-dashboard]]
+[[embedding-dashboards]]
+== Share the dashboard
 
-* <<vega-lite-tutorial-create-your-first-visualizations,Create your first visualization>>
-* <<vega-tutorial-update-kibana-filters-from-vega,Update Kibana filters>>
-* <<reference-for-kibana-extensions,Reference for Kibana extensions>>
-* <<resources-and-examples,Resources and examples>>
+Share the dashboard with a larger audience.
 
-[float]
-[[timelion]]
-=== Create panels with Timelion
+. From the toolbar, click *Share*. 
 
-*Timelion* is a time series data visualizer that enables you to combine independent data sources within a single visualization.
+. Choose one of the following options:
 
-*Timelion* is driven by a simple expression language that you use to:
+* *Embed code* &mdash; Embed the code in a web page. To view the embedded dashboard, users must have {kib} access.
 
-* Retrieve time series data
-* Perform calculation to tease out the answers to complex questions
-* Visualize the results
+* *Permalinks* &mdash; Share a direct link to a {kib} dashboard.
++
+TIP: When you share a link to a dashboard snapshot, select *Short URL*. Snapshot
+URLs are long and can be problematic for Internet Explorer and other
+tools. To create a short URL, you must have write access to {kib}.
 
-[role="screenshot"]
-image::images/timelion.png[Timelion UI]
+* *PDF Reports* &mdash; Generate a PDF report. For more information, refer to <<reporting-getting-started,Reporting>>.
 
-Ready to try out Timelion? For step-by-step tutorials, refer to:
+* PNG Reports &mdash; Generate a PNG report. For more information, refer to <<reporting-getting-started,Reporting>>.
 
-* <<timelion-tutorial-create-time-series-visualizations,Create time series visualizations>>
-* <<timelion-tutorial-create-visualizations-with-mathematical-functions,Create visualizations with mathematical functions>>
-* <<timelion-tutorial-create-visualizations-withconditional-logic-and-tracking-trends,Create visualizations with conditional logic and tracking trends>>
+TIP: To create a link to a dashboard by title, use: +
+`${domain}/${basepath?}/app/dashboards#/list?title=${yourdashboardtitle}`
 
+[discrete]
 [float]
-[[timelion-deprecation]]
-==== Timelion app deprecation
-
-In 7.0 and later, *Timelion* app is deprecated. In 8.0 and later, *Timelion* app is removed from {kib}. To prepare for the removal of *Timelion* app, you must migrate *Timelion* app worksheets to a dashboard.
-
-NOTE: Only *Timelion* app is deprecated. {kib} continues to support *Timelion*
-visualizations in *Dashboard*, *Visualize*, and *Canvas*.
+[[import-dashboards]]
+=== Export the dashboard
 
-To migrate a *Timelion* worksheet to a dashboard:
+Export the dashboard outside of {kib}.
```

Comment:
Dashboards can't be used outside Kibana- exporting is used for automation or backup purposes. Can we remove this section, or move it after the Lens guide?

If you think this section is important, we should have a single-sentence link to the dashboard API:

`Dashboards can be backed up or automated by using the <<dashboard-api>>`